src/databricks/labs/ucx/installer/policy.py

```diff
@@ -68,7 +77,9 @@ def _definition(self, conf: dict, instance_profile: str | None) -> str:
             )
             if instance_profile:
                 policy_definition["aws_attributes.instance_profile_arn"] = self._policy_config(instance_profile)
-        elif self._ws.config.is_azure:  # pylint: disable=confusing-consecutive-elif
+            else:
+                pass
```

Comment:
```suggestion
```
looks redundant

Reply:
this is to avoid pylint [confusing-consecutive-elif](https://pylint.readthedocs.io/en/latest/user_guide/messages/refactor/confusing-consecutive-elif.html)